Documentation/block.md

```diff
@@ -35,6 +35,7 @@ metadata:
 provisioner: rook.io/block
 parameters:
   pool: replicapool
+  clusterName: rook  
```

Comment:
@zcourts in #1614 the `clusterName` has been renamed to `clusterNamespace`. Could you rename that here too? Sounds like consensus is that a comment would still be helpful as well.